### PR TITLE
Update docs to reflect nonstructural building damage change

### PR DESCRIPTION
**Pull request by longshuicy**
Related to pyincore PR: https://github.com/IN-CORE/pyincore/pull/571

### CODE REVIEW

File: manual_jb/content/analyses/non_structural_building_dmg.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ key name | type | name | description
 
 key name | type | parent key | name | description
 --- | --- | --- | --- | ---
-`result` <sup>*</sup> | [`ergo:nsBuildingInventoryDamageVer2`](https://incore.ncsa.illinois.edu/semantics/api/types/ergo:nsBuildingInventoryDamageVer2) | `buildings` | Results | A dataset containing results <br>with both, acceleration sensitivy (AS) related <br>and drift sensitivity (DS) related damage states <br>(format: CSV).
-`damage_result` <sup>*</sup> | [`incore:nsBuildingInventoryDamageSupplement`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:nsBuildingInventoryDamageSupplement) | `buildings` | Results | Information about applied hazard value and fragility<br>(format: JSON).
+`result` <sup>*</sup> | [`ergo:nsBuildingInventoryDamageVer4`](https://incore.ncsa.illinois.edu/semantics/api/types/ergo:buildingDamageVer5) | `buildings` | Results | A dataset containing results <br>(format: CSV).
+`damage_result` <sup>*</sup> | [`incore:buildingDamageSupplement`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:buildingDamageSupplement) | `buildings` | Results | Information about applied hazard value and fragility<br>(format: JSON).
```
Comment (longshuicy) [inline manual_jb/content/analyses/non_structural_building_dmg.md:53]: Note to myself, i need to change it back to incore:nsBuildingInventoryDamageSupplement

File: manual_jb/content/analyses/non_structural_building_dmg.md [hunk 2]
```diff
@@ -2,18 +2,19 @@
 
 **Description**
 
-This analysis computes the non-structural damage to buildings based on a particular hazard. Currently, supported hazard is: **earthquake**.
+This analysis computes the non-structural damage to buildings based on a particular hazard. Currently, supported 
+hazard is: **earthquake** and **flood**.
```
Comment (navarroc) [inline manual_jb/content/analyses/non_structural_building_dmg.md:6]: Should this also include hurricane?

File: manual_jb/content/analyses/non_structural_building_dmg.md [hunk 3]
```diff
@@ -24,8 +25,7 @@ key name | type | name | description
 `result_name` <sup>*</sup> | `str` | Result name | Name of the result dataset.
 `hazard_type` | `str` | Hazard type | Hazard type (earthquake).
 `hazard_id` | `str` | Hazard id | ID of the hazard from the Hazard service.
-`fragility_key_as` | `str` | AS fragility | Acceleration-sensitive fragility key used in mapping dataset.
-`fragility_key_ds` | `str` | DS fragility | Drift-sensitive fragility key used in mapping dataset.
+`fragility_key` | `str` | AS fragility | Acceleration-sensitive fragility key used in mapping dataset.
```
Comment (navarroc) [inline manual_jb/content/analyses/non_structural_building_dmg.md:28]: This should probably be something more generic like:
 Fragility key | Fragility key used in mapping dataset.

**Comment by navarroc** [comment]
Should we change the name in the markdown as well to match the analysis name change and should we also update the analysis list? We could probably leave the markdown file names alone, but we would update analyses.md to be Building Nonstructural Damage instead of Nonstructural building damage?

**Review by navarroc — APPROVED** [review]

**Review by ywkim312 — COMMENTED** [review]
In this PR, non_stuctural_building_dmg.md has been removed and building_nostructural_dmg.md is added. However, the notebook name is still non_structural_building_dmg. This is correct, right?

**Comment by longshuicy** [comment]
> In this PR, non_stuctural_building_dmg.md has been removed and building_nostructural_dmg.md is added. However, the notebook name is still non_structural_building_dmg. This is correct, right?

I further change the jupyter notebook name. Upon approval I can resolve merge confict.

**Review by ywkim312 — APPROVED** [review]

**Pull request merged**